Automattic/wp-calypso
# Update Our Color Palette to the Most Recent Version

* Upgrade `@automattic/color-studio` to the most recent version (that updates to Blue and Celadon: Automattic/color-studio#37).
* Tweak the SVG audit script to properly catch all images that might use non-standard color values.
* Update affected images so that they only use standardized color values.
* Update any outdated static color values found in the codebase.

#### Testing Instructions

*Local Testing Only*

* Pull the changes:
    * `git checkout update/colors`
    * `yarn`
* Run the SVG color audit script and make sure it doesn’t report any specific paths.
    * `node bin/audit-svg-colors`
* Preview the images listed by the command below:
    * `git diff trunk --name-only | grep 'svg$'`
    * Make sure none of them exhibits any visual regressions or errors that would prevent them from displaying.
